test/test_model.py

```diff
@@ -102,73 +62,152 @@ def test_get_relation(self):
         self.assertEqual(dead_rel.data[self.model.unit], {})
         self.assertIsNone(self.model.get_relation('db0'))
         self.assertIs(self.model.get_relation('db1'), db1_4)
-        with self.assertRaises(op.model.TooManyRelatedApps):
+        with self.assertRaises(op.model.TooManyRelatedAppsError):
             self.model.get_relation('db2')
 
+        self.assertEqual(fake_script_calls(self), [
+            ['relation-ids', 'db1', '--format=json'],
+            ['relation-list', '-r', '4', '--format=json'],
+            ['relation-list', '-r', '7', '--format=json'],
+            ['relation-get', '-r', '7', '-', 'myapp/0', '--format=json'],
+            ['relation-ids', 'db0', '--format=json'],
+            ['relation-ids', 'db2', '--format=json'],
+            ['relation-list', '-r', '5', '--format=json'],
+            ['relation-list', '-r', '6', '--format=json']])
```

Comment:
If we finalize with this, the formatting becomes more even and means we can add and remove lines without reformatting existing lines:
```
self.assertEqual(fake_script_calls(self), [
        [...],
        [...],
])
```